tigerpath/majors_and_certificates/majors/SLA_2018.json

```diff
@@ -0,0 +1,131 @@
+{
+  "type": "Major",
+  "name": "Slavic Languages and Literatures",
+  "code": "SLA",
+  "degree": "AB",
+  "year": 2018,
+  "explanation": "A minimum of eight departmental courses is required. Four upper-level (200 and above) courses must be within the department, two of which must be from the core survey courses (SLA 219, 220, 221); the other four courses may be from cognate areas depending on the student's particular interests. For example, if the major field of concentration is 19th-century prose, the program might include courses from French or German literature. Students with a strong interest in Russian and Soviet studies might take area courses in the Program in Russian and Eurasian Studies such as Russian history, politics, anthropology, or sociology.",
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-slavic-languages-and-literatures",
+    "https://slavic.princeton.edu/undergraduate-program/program-requirements"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Michael A. Wachtel",
+      "email": "wachtel@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Elena Fratto",
+      "email": "efratto@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "RUS 101, RUS 102, RUS 105, and RUS 107 (RUS 105R, 107R) – or placement equivalent.",
+      "course_list": [
+        "RUS 101",
+        "RUS 102",
+        "RUS 105",
+        "RUS 107"
+      ]
+    },
+    {
+      "name": "Required Courses",

```

Comment:
Do these courses not count as part of the 8 departmental courses?
Are they actually intended to be separate? I'm having a hard time understanding the way they're phrasing it on the department webpage.

Reply:
I think they are separate since the department page does not explicitly state that RUS 207 and the advanced language is part of the four upper-level courses, which it does for the core survey courses. The required courses and the departmentals are also mentioned separately in the UA.